standalone-metastore/metastore-tools/pom.xml

```diff
@@ -81,6 +81,10 @@
             <groupId>org.apache.curator</groupId>
             <artifactId>curator-client</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
Jettison dependency that needed to be excluded is coming from tez-api as mentioned in my previous comment and not hive-common please exclude from there.
[depend.txt](https://github.com/apache/hive/files/11302939/depend.txt)

Please refer to the attached dependency tree to verify.